src/server/events.js

```diff
@@ -0,0 +1,36 @@
+var utils = require('./utils.js')
+var exports = module.exports = {};
+
+exports.create = function(newEvent){
```

Comment:
Yes, I have a branch im working on that will handle converting all ES5 to ES6 changes